llpc/test/shaderdb/PipelineVsFs_RelocConst.pipe

```diff
@@ -0,0 +1,103 @@
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -use-relocatable-shader-elf -o test_pipeline_out_relocconst.elf %gfxip %s && llvm-objdump -triple=amdgcn -mcpu=gfx900 -d test_pipeline_out_relocconst.elf | FileCheck -check-prefix=SHADERTEST %s && rm test_pipeline_out_relocconst.elf
+; SHADERTEST-LABEL: 0000000000000000 _amdgpu_vs_main:
+; SHADERTEST: s_mov_b32 s[[RELOREG:[0-9]+]], 12 //{{.*}}
+; SHADERTEST: s_load_dwordx4 s[{{.*}}:{{.*}}], s[{{.*}}:{{.*}}], s[[RELOREG]] //{{.*}}
+; SHADERTEST: {{[0-9A-Za-z]+}} _amdgpu_ps_main:
+; END_SHADERTEST
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -use-relocatable-shader-elf -v %gfxip %s | FileCheck -check-prefix=SHADERTEST1 %s
+; SHADERTEST1-LABEL: {{^// LLPC}} pipeline patching results
+; SHADERTEST1: %[[RELOCONST:[0-9]+]] = call i32 @llvm.amdgcn.reloc.constant(metadata !{{.*}})
```

Comment:
We should check that the name generated for the symbol is correct.

Reply:
The name of the relocation symbol is not expected to be in the final linked elf so this check is not possible. However the test does attempt to check if the correct offset is written in the linked result.

Reply:
At this point you are not checking the final linked elf.  It is looking at the llvm-ir after patching.  The llvm-ir should have the symbol name in the metadata that is a parameter to the builtin.  We want this part of the test to say that the patching passes did the right thing.

Reply:
Done.